hera_qm/ant_metrics.py

```diff
@@ -10,6 +10,32 @@
 from hera_qm import utils
 
 
+ant_metrics_list = {'ant_metrics_meanVij': 'Mean of the absolute value of all '
```

Comment:
Yes, I will add comments explaining these. Actually, I wonder if it would be cleaner to make a function, `get_metrics_dict` or something similar, which just returns this dict. That way I can add a docstring, and it will be a little more clear when things are coming from. Would that make more sense?